src/content/blog/2024_lre.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Adding Layerwise Richardson Extrapolation in Mitiq"
+author: Purva Thakre
+day: 28
+month: 11
+year: 2024
+tags:
+    - Mitiq
+---
+
+Layerwise Richardson Extrapolation (LRE) [^1] is a multivariate extension of Zero Noise Extrapolation (ZNE)
+where the noise of each layer is treated as an independent variable.
```

Comment:
Adding a little lead-in here would be good. This is the UF blog, so people might need a little easing in to QEM.